### DISCUSSION TITLE
replace "r n b q k b n r" with other characters.

### DISCUSSION THREAD

**Discussion by vwallajabad** [niklasf/python-chess#898] [Q&A]
Hello,
I was wondering if there is a function or and other way to replace the default ```r n b q k b n r```  with anything else. For example, if I want to use emoji's as chess pieces is that possible using the current chess package?

**Comment by niklasf** [comment]
Hi. Sure, translating symbols like that can be done without any chess logic:

```python
trans = str.maketrans({"a": "foo", "b": "bar", "c": "baz"})
"abc".translate(trans) # foobarbaz
```

**Reply by vwallajabad** [reply]
Could someone create this as a feature built in to the package itself. I could see this helping a lot of people to create custom pieces. 

Or should I put this as an issue and create a pull request to add the feature?

**Reply by niklasf** [reply]
`str.translate` really seems concise enough:

```python
import chess

trans = str.maketrans({"P": "♙", "p": "♟"})

board = chess.Board()
print(str(board).translate(trans))
```

```
r n b q k b n r
♟ ♟ ♟ ♟ ♟ ♟ ♟ ♟
. . . . . . . .
. . . . . . . .
. . . . . . . .
. . . . . . . .
♙ ♙ ♙ ♙ ♙ ♙ ♙ ♙
R N B Q K B N R
```

**Reply by vwallajabad** [reply]
Oh ok. I was trying ```board.translate(trans)``` instead of ```str(board).translate(trans) ```. I dont know why I did not think of that. Thanks!
